config_defaults/subtests/docker_cli/build.ini

```diff
@@ -98,8 +98,7 @@ docker_registry_user = stackbrew
 docker_repo_name = centos
 docker_repo_tag = latest
 postproc_cmd_csv = positive(),
-                   rx_out('^Schazam!$'),
-                   rx_out('^foobar$'),
+                   rx_out('^It works!$'),
```

Comment:
TIL that I've had the `git_path` test disabled in my setup since April 2016. Whoops. Anyhow, tested just now and it works.